alien-module/prototypes/item/alien-warfare.lua

```diff
@@ -62,7 +64,7 @@ if (settings.startup["alien-module-hyper-ammo-enabled"].value) then
 								},
 								{
 									type = "damage",
-									damage = { amount = 5 + 0.2 * i, type = "physical" }
+									damage = { amount = 14 + 0.2 * i, type = "physical" }
```

Comment:
Should remain at 5

Reply:
I think the alien ammunition (5 + 0.2 * i) is too bad compared to the mid game armor piercing ammunition (14). It should be classified between armor-piercing ammunition (14) and ammunition mixed with uranium (24).